### ISSUE TITLE
question about  pre-QC data

### ISSUE THREAD

**Issue by wangpenhok** [molgenis/STEMI-scRNA-seq#29]
hello, thanks for your kind share of the expression data! I really appreciate the work you have done, which deepened our recognition of immune landscape during STEMI.  I want to include some data into my own study,  but I came across a question about the data in part C, the pre-QC STEMI non-normalized counts: 
    stemi_unfiltered_barcodes.tsv.gz
    stemi_unfiltered_features_raw.tsv.gz
    stemi_unfiltered_matrix_raw.mtx.gz

I am wondering if these data are exactly the same with those from `raw_feature_matrix` directory of cellRanger output. Could you please help me on this question? And  if not, I am wondering if it is possible to share the `raw_feature_matrix` contents with me.

Looking forward to your reply ~

Sincerely

**Comment by mcamenc** [comment]
Hi!

I would also like to know if it could be possible to share the raw data for each stemi patient, as I guess the pre-QC STEMI non-normalized counts have all samples merged?

Many thanks and regards.

**Comment by royoelen** [comment]
These raw count matrices have been merged across the 10x sample pools that we included in our experiments (no filtering). The metadata however contains a 'lane' column (the sample pool) which allows you to split up again the counts coming from specific lanes. This lane is also present as part of the cell barcode/ID. If you also need the other files such as the bams, you unfortunately need to request access to the raw data via the paragraph at the bottom of this page https://eqtlgen.org/sc/datasets/blokland2024-dataset.html
